ddp-workspace/projects/ddp-sdk/src/lib/services/serviceAgents/userDeletionServiceAgent.service.ts

```diff
@@ -0,0 +1,18 @@
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs';
+import { take } from 'rxjs/operators';
+
+import { UserServiceAgent } from './userServiceAgent.service';
+
+@Injectable()
+export class UserDeletionServiceAgent extends UserServiceAgent<void> {
+  public deleteUser(userGuid: string): Observable<void> {
+    const url = `${this.configuration.backendUrl}/pepper/v1/user/${userGuid}`;
```

Comment:
I believe deleteObservable() is already including part of this path. You might just need `/user/${userGuid}` , but take closer look.

Reply:
The way how user deletion works is that only proxy user is allowed to delete it's managed users. Therefore we have to manually pass the `userGuid` since underlying `UserServiceAgent` dynamically substitutes current `userGuid` with either from proxy user or a participant if one is set.
I just thought that it would be more logical to extend the `UserServiceAgent` since we are dealing with user entity.

Reply:
Sure, this sounds reasonable.